src/lib/processor.py

```diff
@@ -1,7 +1,10 @@
 import do
 import actions
 from exceptions import *
```

Comment:
OK I think it does, I've raised exceptions before that way, but I can see how its not a good practice